types/file-saver/test/file-saver-tests.ts

```diff
@@ -0,0 +1,73 @@
+import { saveAs, FileSaverOptions } from 'file-saver';
```

Comment:
Why `saveAs` is using `import` which is different from `saveAsReq` here?

Reply:
To test named imports (`import { saveAs } from 'file-saver'`) vs CommonJS style imports (`import saveAs = require('file-saver')`). The former works in ES6 module environments, the latter with node requires.

Reply:
`file-saver` is not `ES`, so I don't think we need to test that?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#esmoduleinteropallowsyntheticdefaultimports

Reply:
It's not, but it still exposes a `saveAs` export, hence that's why I'm testing it.

Reply:
And by the way, this is not a default export that I'm testing, a default export would be: `import saveAs from 'file-saver'`.

Reply:
> it still exposes a saveAs export

You mean `saveAs.saveAs`, right? It is still commonjs, not ES `export`.

Reply:
It is, but it's compatible with ESM at least with TS and WebPack.